src/anyvar/utils/liftover_utils.py

```diff
@@ -0,0 +1,292 @@
+    :param position: A SequenceLocation start or end position. Can be a `list` or an `int`.
+
+    :return: A lifted-over position. Type (`list` or `int`) will match that of `position`
+    """
+    # Handle int positions
+    if isinstance(position, int):
+        return _convert_coordinate(converter, chromosome, position)
+
+    # Handle Range (list) positions
+    lower_bound, upper_bound = position
+    lower_bound = (
+        _convert_coordinate(converter, chromosome, lower_bound) if lower_bound else None
+    )
+    upper_bound = (
+        _convert_coordinate(converter, chromosome, upper_bound) if upper_bound else None
+    )
+    return [lower_bound, upper_bound]
+
+
+def get_liftover_variant(
+    variant_object: dict, seqrepo_dataproxy: _DataProxy
+) -> VrsVariation:
+    """Liftover a variant from GRCh37 or GRCH38 into the opposite assembly, and return the converted variant as a VrsObject.
+    If liftover is unsuccessful, raise an Exception.
+
+    :param variant_object: A dictionary representation of a `VrsVariation`.
+    :param seqrepo_dataproxy: A `SeqrepoDataproxy` instance.
+    :return: The converted variant as a `VrsObject`.
+    :raises:
+        - `MalformedInputError`:  If the `variant_object` is empty or otherwise falesy
+
+        - `UnsupportedVariantLocationTypeError`: If the variant lacks a refget accession, start position or end position
+
+        - `UnsupportedReferenceAssemblyError`: If the variant's accession was not found in any supported assembly
+
+        - `AmbiguousReferenceAssemblyError`: If the variant's accession was found in multiple supported assemblies
+
+        - `ChromosomeResolutionError`: If unable to resolve variant's chromosome
+
+        - `CoordinateConversionError`: If unable to lift over the variant's start and/or end position(s)
+
+        - `AccessionConversionError`: If unable to lift over the variant's refget accession
+    """
+    if not variant_object:
+        raise MalformedInputError
+
+    # Get variant start position, end position, and refget accession - liftover is currently unsupported without these
+    refget_accession = (
+        variant_object.get("location", {})
+        .get("sequenceReference", {})
+        .get("refgetAccession")
+    )
+    start_position = variant_object.get("location", {}).get("start")
+    end_position = variant_object.get("location", {}).get("end")
+    if not refget_accession or not start_position or not end_position:
+        raise UnsupportedVariantLocationTypeError
+
+    # Determine which assembly we're converting from/to
+    prefixed_accession = f"ga4gh:{refget_accession}"
+
+    grch37 = "GRCh37"
+    grch38 = "GRCh38"
+
+    # This is required for `get_from_and_to_assemblies` and `get_chromosome_from_aliases`
+    # See function documentation for more details
+    accession_aliases = {
```

Comment:
I think you can simplify the next few lines down to something like this

```python
    if identifiers := seqrepo_dataproxy.translate_sequence_identifier(prefixed_accession, grch37):
        from_genome, to_genome = Genome.HG19, Genome.HG38
    elif identifiers := seqrepo_dataproxy.translate_sequence_identifier(prefixed_accession, grch38):
        from_genome, to_genome = Genome.HG38, Genome.HG19
    else:
        msg = f"Unable to get reference sequence ID for {prefixed_accession}"
        raise UnsupportedReferenceAssemblyError(msg)
    from bioutils.accessions import chr22XY
    chromosome = chr22XY(identifiers[0].split(":")[1])
    converter = Converter(from_genome, to_genome)
```

Reply:
Also, minor point, but I think we should check the <from 38, to 37> case first... in the long term it's probably the more likely form of input